sublime-package.json

```diff
@@ -181,4 +191,4 @@
       }
     ]
   }
-}
+}
```

Comment:
removed trailing newline? :)

Reply:
I saw that when I went to bed, 
but was on my phone 🙂

Reply:
```suggestion
}

```

Reply:
... fixed, found a way to push small commits with a phone 🙃